include/eve/detail/function/simd/common/friends.hpp

```diff
@@ -152,10 +153,7 @@ namespace eve::detail
     }
     else
     {
-      auto zipped = kumi::zip(v.storage(),w.storage());
-      return kumi::fold_right ( [](auto acc , auto e) { return acc && ( get<0>(e) == get<1>(e) ); }
-                              , zipped, true
-                              );
+      return v.storage() == w.storage();
```

Comment:
What does this actually do?
I mean, what would `kumi::tuple::operator==` return here?

Reply:
What type will it be, for example, for `wide<pair<int, double>`

Reply:
The == in kumi just do what this complicated zip does in a simpler way cause it has access to the element via ...
The type return is what `as_logical_t<wide<pair<int,double>>` does, a `logical<wide<int>>`